modules/nf-core/orthofinder/tests/main.nf.test.snap

```diff
@@ -1,322 +1,18 @@
-                "OG0000224.fa",
-                "OG0000225.fa",
-                "OG0000226.fa",
-                "OG0000227.fa",
-                "OG0000228.fa",
-                "OG0000229.fa",
-                "OG0000230.fa",
-                "OG0000231.fa",
-                "OG0000232.fa",
-                "OG0000233.fa",
-                "OG0000234.fa",
-                "OG0000235.fa",
-                "OG0000236.fa",
-                "OG0000237.fa",
-                "OG0000238.fa",
-                "OG0000239.fa",
-                "OG0000240.fa",
-                "OG0000241.fa",
-                "OG0000242.fa",
-                "OG0000243.fa",
-                "OG0000244.fa",
-                "OG0000245.fa",
-                "OrthoFinder_graph.txt",
-                "Orthogroups.GeneCount.tsv",
-                "Orthogroups.tsv",
-                "Orthogroups.txt",
-                "Orthogroups_SingleCopyOrthologues.txt",
-                "Orthogroups_SpeciesOverlaps.tsv",
-                "Orthogroups_UnassignedGenes.tsv",
-                "OrthologuesStats_Totals.tsv",
-                "OrthologuesStats_many-to-many.tsv",
-                "OrthologuesStats_many-to-one.tsv",
-                "OrthologuesStats_one-to-many.tsv",
-                "OrthologuesStats_one-to-one.tsv",
-                "SequenceIDs.txt",
-                "SimpleTest.phy",
-                "SimpleTest.phy_fastme_stat.txt",
-                "SimpleTest.tre",
-                "Species0.fa",
-                "Species0.fa",
-                "Species1.fa",
-                "SpeciesIDs.txt",
-                "SpeciesTree_Gene_Duplications_0.5_Support.txt",
-                "SpeciesTree_rooted.txt",
-                "SpeciesTree_rooted_ids.txt",
-                "SpeciesTree_rooted_node_labels.txt",
-                "SpeciesTree_unrooted.txt",
-                "SpeciesTree_unrooted_ids.txt",
-                "Statistics_Overall.tsv",
-                "Statistics_PerSpecies.tsv",
-                "candidatus_portiera_aleyrodidarum.tsv",
-                "candidatus_portiera_aleyrodidarum.tsv",
-                "candidatus_portiera_aleyrodidarum__v__sarscov2.tsv",
-                "clusters_OrthoFinder_I1.5.txt",
-                "clusters_OrthoFinder_I1.5.txt_id_pairs.txt",
-                "diamondDBSpecies0.dmnd",
-                "diamondDBSpecies0.dmnd",
-                "diamondDBSpecies1.dmnd",
-                "sarscov2.tsv",
-                "sarscov2.tsv",
-                "sarscov2__v__candidatus_portiera_aleyrodidarum.tsv",
-                "test_search_results.txt.gz"
-            ],
             [
                 "Statistics_PerSpecies.tsv:md5,984b5011a34d54527fe17896bfa36a2d",
-                "SpeciesTree_Gene_Duplications_0.5_Support.txt:md5,8b7a673e2e8b6d1aeb697f2bb88afa18",
-                "SpeciesTree_rooted.txt:md5,4d5ea525feebe479fca0c0768271ba81"
+                "SpeciesTree_rooted.txt:md5,4d5ea525feebe479fca0c0768271ba81",
```

Comment:
I think these will stop moving around if we used `stable_files.toSorted()`.

Reply:
Great, yes, this solved it